modules/vpc-sc/README.md

```diff
@@ -107,65 +99,61 @@ module "test" {
   access_policy = "12345678"
   access_levels = {
     a1 = {
-      combining_function = null
-      conditions = [{
-        members       = ["user:user1@example.com"], ip_subnetworks = null,
-        negate = null, regions = null, required_access_levels = null
-      }]
+      conditions = [
+        { members = ["user:user1@example.com"] }
+      ]
     }
     a2 = {
-      combining_function = null
-      conditions = [{
-        members       = ["user:user2@example.com"], ip_subnetworks = null,
-        negate = null, regions       = null, required_access_levels = null
-      }]
+      conditions = [
+        { members = ["user:user2@example.com"] }
+      ]
+    }
+  }
+  egress_policies = {
+    # allow writing to external GCS bucket from a specific SA
+    gcs-sa-foo = {
+      from = {
+        identities = [
+          "serviceAccount:foo@myproject.iam.gserviceaccount.com"
+        ]
+      }
+      to = {
+        operations = [{
+          method_selectors = ["*"]
+          service_name = "storage.googleapis.com"
+        }]
+        resources = ["projects/123456789"]
+      }
+    }
+  }
+  ingress_policies = {
+    # allow management from external automation SA
+    sa-tf-test = {
+      from = {
+        identities = [
+          "serviceAccount:test-tf@myproject.iam.gserviceaccount.com",
+        ],
+        source_access_levels = ["*"]
```

Comment:
What about adding in the example an Access level for company proxy IP ranges and use it here to limit the ingress policy to company proxy ips?

The example would suggest a more secure solution (automation SA has high roles that is better secure).

Reply:
Absolutely. Do you mind adding it after we merge?